components/textfield/stories/textarea.stories.js

```diff
@@ -0,0 +1,144 @@
+	chromatic: { disableSnapshot: true }
+};
+
+/**
+ * A text area in a disabled state shows that an input field exists, but is not available in that circumstance. This can be used to maintain layout continuity and communicate that a field may become available later.
+*/
+export const Disabled = Template.bind({});
+Disabled.tags = ["!dev"];
+Disabled.args = {
+	isDisabled: true
+};
+Disabled.parameters = {
+	chromatic: { disableSnapshot: true }
+};
+
+/**
+ * A text area can be marked as having an error to show that a value needs to be entered in order to move forward or that a value that was entered is invalid. If an error exists, the error icon always overrides the validation icon.
+*/
+export const Invalid = Template.bind({});
+Invalid.tags = ["!dev"];
+Invalid.args = {
+	isInvalid: true
+};
+Invalid.parameters = {
+	chromatic: { disableSnapshot: true }
+};
+
+/**
+ * A text area can have [help text](/docs/components-help-text--docs) below the field to give extra context or instruction about what a user should input in the field. The help text area has two options: a description and an error message. The description communicates a hint or helpful information, such as specific requirements for correctly filling out the field. The error message communicates an error for when the field requirements aren’t met, prompting a user to adjust what they had originally input.
+ *
+ * Instead of placeholder text, use the help text description to convey requirements or to show any formatting examples that would help user comprehension. Putting instructions for how to complete an input, requirements, or any other essential information into placeholder text is not accessible.
+*/
+export const HelpText = HelpTextOptionsTextArea.bind({});
+HelpText.tags = ["!dev"];
+HelpText.parameters = {
+	chromatic: { disableSnapshot: true }
+};
+
+export const Quiet = Template.bind({});
+Quiet.tags = ["!dev"];
+Quiet.args = {
+	isQuiet: true,
+};
+Quiet.parameters = {
+	chromatic: { disableSnapshot: true }
+};
+
+/**
+ * Text area has a read-only option for when content in the disabled state still needs to be shown. This allows for content to be copied, but not interacted with or changed.
+*/
+export const Readonly = Template.bind({});
+Readonly.tags = ["!dev"];
+Readonly.args = {
+	isReadOnly: true,
+	value: "Adipisicing dolor quis ad non ad ipsum irure ullamco."
+};
+Readonly.parameters = {
+	chromatic: { disableSnapshot: true }
+};
+Readonly.storyName = "Read-only";
+
+/**
+ * Side labels are most useful when vertical space is limited.
+*/
+export const SideLabel = Template.bind({});
+SideLabel.tags = ["!dev"];
+SideLabel.args = {
+	labelPosition: "side",
+	labelText: "Comments",
+	value: "Qui nulla cupidatat do ex laborum ipsum et culpa reprehenderit dolore.",
+	displayCounter: true,
+	characterCount: 50,
+	helpText: "Example help text. Lorem ipsum dolor sit amet."
+};
+SideLabel.parameters = {
+	chromatic: { disableSnapshot: true }
+};
+
+
```

Comment:
It looks like [the docs site](https://opensource.adobe.com/spectrum-css/textarea.html) has the sizing for textarea too- could you add a story for that? Or was it intentional to not document it?